### PR TITLE
Fix memcache configuration in Swift   to include all nodes in cluster

### PR DESCRIPTION
**Pull request by ryshah**
This will fix issue #1130

### CODE REVIEW

File: roles/swift-proxy/templates/etc/swift/proxy-server.conf [hunk 1]
```diff
@@ -83,9 +83,6 @@ use = egg:swift#container_sync
 [filter:dlo]
 use = egg:swift#dlo
 
-[filter:staticweb]
-use = egg:swift#staticweb
-
```
Comment (ryshah) [inline roles/swift-proxy/templates/etc/swift/proxy-server.conf]: There was another entry for staticweb filter in conf file

**Comment by dlundquist** [comment]
ok to test

**Comment by paulczar** [comment]
ok to test

**Comment by ryshah** [comment]
@j2sol @paulczar @dlundquist  Can we restart tests again so these changes can be merged?

**Comment by j2sol** [comment]
retest

**Comment by j2sol** [comment]
You may need to do an empty git commit --amend and force push to trigger another test attempt.

**Comment by ryshah** [comment]
@j2sol That worked - thanks!

**Pull request merged**